javalib/src/main/scala/java/lang/Math.scala

```diff
@@ -30,8 +31,16 @@ object Math {
   @inline def abs(a: scala.Long): scala.Long = if (a < 0) -a else a
 
   // Wasm intrinsics
-  @inline def abs(a: scala.Float): scala.Float = js.Math.abs(a).toFloat
-  @inline def abs(a: scala.Double): scala.Double = js.Math.abs(a)
+  @inline def abs(a: scala.Float): scala.Float = linkTimeIf(LinkingInfo.targetPureWasm) {
+      throw new AssertionError("Not implemented.")
+    } {
+      js.Math.abs(a).toFloat
+    }
+  @inline def abs(a: scala.Double): scala.Double = linkTimeIf(LinkingInfo.targetPureWasm) {
+      throw new AssertionError("Not implemented.")
```

Comment:
```suggestion
  @inline def abs(a: scala.Double): scala.Double = linkTimeIf(LinkingInfo.targetPureWasm) {
    if (a == -0.0) 0.0
    else if (a < 0.0) -a
    else a
```
Same for the `Float` overload.